adminSiteServer/testPageRouter.tsx

```diff
@@ -257,7 +257,7 @@ async function propsFromQueryParams(
 
     const charts: ChartItem[] = (await query.getMany()).map((c) => ({
         id: c.id,
-        slug: c.config.slug,
+        slug: c.config.slug!,
```

Comment:
Without knowing how we actually use these objects throughout the codebase, shouldn't this be `c.config.slug ?? ""` instead? Otherwise we might accidentally call a method on `undefined`.